devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDeviceVariableTreeNames.h

```diff
@@ -0,0 +1,56 @@
+#ifndef BIPEDAL_LOCOMOTION_fRAMEWORK_YARP_ROBOT_LOGGER_DEVICE_VTN_H
+#include <iostream>
+
+#define BIPEDAL_LOCOMOTION_fRAMEWORK_YARP_ROBOT_LOGGER_DEVICE_VTN_H
+
+#define TREE_DELIM                              "::"
+
+#define ROBOT_RT_ROOT_NAME                      "robot_realtime"
+
+
+#define JOINT_STATE_POSITIONS_NAME              "joints_state::positions"
+#define JOINT_STATE_VELOCITIES_NAME             "joints_state::velocities"
+#define JOINT_STATE_ACCLERATIONS_NAME           "joints_state::accelerations"
+#define JOINT_STATE_TORQUES_NAME                "joints_state::torques"
+
+#define MOTOR_STATE_POSITIONS_NAME              "motors_state::positions"
+#define MOTOR_STATE_VELOCITIES_NAME             "motors_state::velocities"
+#define MOTOR_STATE_ACCELERATIONS_NAME          "motors_state::acclerations"
+#define MOTOR_STATE_CURRENTS_NAME               "motors_state::currents"
+#define MOTOR_STATE_PWM_NAME                    "motors_state::PWM"
+
+#define MOTOR_STATE_PIDS_NAME                   "PIDs"
+
+#define FTS_NAME                                "FTs"
```

Comment:
You can use a ``constexpr std::string_view`` to avoid converting it to string manually later. See https://en.cppreference.com/w/cpp/string/basic_string_view

Reply:
(``string_view`` cannot be summed either)